state/migration_export.go

```diff
@@ -1739,7 +1739,8 @@ func (e *exporter) secrets() error {
 			Created:    rev.CreateTime,
 			Updated:    rev.UpdateTime,
 			ExpireTime: rev.ExpireTime,
-			BackendId:  rev.BackendId,
```

Comment:
Probably error then if this isn't implemented yet.

Reply:
I just added it